src/rules/function-url-quotes/__tests__/index.js

```diff
@@ -426,3 +428,14 @@ testRule(rule, {
     column: 21,
   } ],
 })
+
+testRule(rule, {
```

Comment:
Can we get the same three tests (slightly tweaked) for `config: [ "never", { except: ["empty"] } ],` please?